test/parallel/test-vm-basic.js

```diff
@@ -128,3 +128,23 @@ const vm = require('vm');
              'Received type object'
   });
 });
+
+// vm.getWrappedFunction
+{
+  assert.strictEqual(
+    vm.getWrappedFunction('console.log("Hello, World!");'),
+    'function (exports, require, module, __filename, __dirname)'
+    + '{\nconsole.log("Hello, World!");\n}'
+  );
+
+  assert.doesNotThrow(vm.getWrappedFunction('return;'))
+
+  common.expectsError(() => {
+    vm.getWrappedFunction(
+      '});\n\n(function() {\nconsole.log(1);\n})();\n\n(function() {'
+    );
+  }, {
+    type: SyntaxError,
+    message: 'Unexpected token }'
+  });
+}
```

Comment:
I thought you had more tests :)

Reply:
Yeah, I commented out these as well, for now. Now that the existing tests all pass, will focus on covering as many cases as possible with tests.

Reply:
Yeah, I commented out these as well, for now. Now that the existing tests all pass, will focus on covering as many cases as possible with tests.

Reply:
Yeah, I commented out these as well, for now. Now that the existing tests all pass, will focus on covering as many cases as possible with tests.